fundingmanager.go

```diff
@@ -1241,6 +1243,7 @@ func (f *fundingManager) handleFundingAccept(fmsg *fundingAcceptMsg) {
 		MinHTLC:          msg.HtlcMinimum,
 		MaxAcceptedHtlcs: msg.MaxAcceptedHTLCs,
 		CsvDelay:         msg.CsvDelay,
+		NumConfs:         msg.MinAcceptDepth,
```

Comment:
We can just check the value here instead of making it part of the constraints.

Reply:
Fixed.